cli/src/cli_util.rs

```diff
@@ -1012,7 +1012,12 @@ impl WorkspaceCommandHelper {
         let immutable = revset_util::parse_immutable_expression(&self.revset_parse_context())?;
         let mut expression = self.attach_revset_evaluator(immutable)?;
         expression.intersect_with(&to_rewrite_revset);
-        if let Some(commit_id) = expression.evaluate_to_commit_ids()?.next() {
+
+        let mut commit_id_iter = expression.evaluate_to_commit_ids().map_err(|e| {
+            config_error_with_message("Invalid `revset-aliases.immutable_heads()`", e)
+        })?;
```

Comment:
(Optional) can you also add a similar check to the `parse_*()` result?

Reply:
Should I just do the whole chain then? (meaining also attach_revset_evaluator)

Reply:
No we can't. They returns different error types, and `attach_revset_evaluator()` has its error context (= "revse.short-prefixes".)

Reply:
👍🏻